gen/size.go

```diff
@@ -79,6 +79,8 @@ func (s *sizeGen) Execute(p Elem) error {
 		return nil
 	}
 
+	s.p.comment("Msgsize returns the number of bytes occupied by the serialized message")
```

Comment:
Would you mind re-wording this slightly to indicate that `Msgsize()` is an upper-bound estimate, and not the exact number of bytes?